collector/nats/dashboards/server/main.tf

```
@@ -0,0 +1,223 @@
+terraform {
+  required_providers {
+    lightstep = {
+      source  = "lightstep/lightstep"
+      version = "~> 1.76.0"
+    }
+  }
+  required_version = ">= v1.0.11"
+}
+
+variable "cloud_observability_project" {
```

```suggestion
variable "lightstep_project" {
```

For now, the name must be `lightstep_project`. Will apply the change across the repo when we can make this change.In the meantime... happy with the description of the var.

---

I applied this in the 3 places impacted.